packages/monaco/src/browser/monaco-snippet-suggest-provider.ts

```diff
@@ -35,6 +35,8 @@ export class MonacoSnippetSuggestProvider implements monaco.modes.ISuggestSuppor
     }
 
     resolveCompletionItem(_: monaco.editor.ITextModel, __: monaco.Position, item: monaco.modes.ISuggestion): monaco.modes.ISuggestion {
+        const word = _.getWordAtPosition(__);
```

Comment:
please rename to proper names, underscore means that parameters are not used

used variables should follow naming guidelines: https://github.com/theia-ide/theia/wiki/Coding-Guidelines#whole-words-names